skbio/stats/composition.py

```diff
@@ -2058,3 +2060,489 @@ def dirmult_ttest(
+                _lme_coef = float(summary.tables[1]["Coef."][var_name])
+                _lme_25 = float(summary.tables[1]["[0.025"][var_name])
+                _lme_975 = float(summary.tables[1]["0.975]"][var_name])
+
+                individual_results = {
+                    FEATUREID: response_var,
+                    COVARIATE: var_name,
+                    LOG2FC: _lme_coef,
+                    CI25: _lme_25,
+                    CI975: _lme_975,
+                    PVALUE: results.pvalues[var_name],
+                }
+            except Exception as e:
+                if type(e) is not ValueError:
+                    print("Something went really wrong, details below: ")
+                    print(type(e))
+                    print(e)
+
+                    individual_results = {
+                        FEATUREID: response_var,
+                        COVARIATE: var_name,
+                        LOG2FC: np.nan,
+                        CI25: np.nan,
+                        CI975: np.nan,
+                        PVALUE: np.nan,
+                    }
+
+                else:
+                    measures = {
+                        LOG2FC: np.nan,
+                        CI25: np.nan,
+                        CI975: np.nan,
+                    }
+                    mapping = {
+                        LOG2FC: _lme_coef,
+                        CI25: _lme_25,
+                        CI975: _lme_975,
+                    }
+
+                    if len(summary.tables) >= 2:
+                        table = summary.tables[1]
+
+                        for key_final, key_table in mapping.items():
+                            if key_table in table and var_name in table[key_table]:
+                                measures[key_final] = table[key_table][var_name]
+
+                    individual_results = {
+                        "FeatureID": response_var,
+                        "Covariate": var_name,
+                        "pvalue": results.pvalues[var_name],
+                    }
+
+                    individual_results.update(measures)
+
+            output.append(individual_results)
+
+    return (output, submodels, _covariate_list)
+
+
+def _obtain_dir_table(data, pseudocount, rng):
+    if data.shape[1] > 1:
+        posterior = [
+            rng.dirichlet(data.values[i] + pseudocount) for i in range(data.shape[0])
+        ]
+        dir_table = pd.DataFrame(
+            posterior, index=data.index, columns=data.columns
+        ).apply(clr, axis=1)
+        dir_table = pd.DataFrame(
+            dir_table.tolist(), index=dir_table.index, columns=data.columns
+        )
+    else:
+        posterior = rng.dirichlet(data.values.flatten() + pseudocount)
+        dir_table = np.log(posterior)
+        dir_table = pd.DataFrame(dir_table, index=data.index, columns=data.columns)
+
+    return dir_table
+
+
+def dirmult_lme(
+    formula,
+    data,
+    metadata,
+    groups=None,
+    reml=True,
```

Comment:
Let's remove this and see users can still pass in `reml` (aka check if the tests still pass)